gallery/__init__.py

```diff
@@ -128,6 +128,8 @@ def index():
 @auth.oidc_auth('default')
 @gallery_auth
 def view_upload(auth_dict: Optional[Dict[str, Any]] = None):
+    member_lock = session['userinfo'].get('member-lock')
+
```

Comment:
all the places where you're doing this and not using member_lock anywhere but the render_template, why not skip the variable and just do `session....get...` in the render_template args?